From: Alexander Larsson <alexl@redhat.com>
Date: Fri, 20 Dec 2019 16:47:28 +0100
Subject: xml: Avoid fancy quotes

This breaks the build on older versions of gdbus-codegen, such
as the one in Ubuntu xenial.

Origin: upstream, 1.7.0, commit:fb020e171fb151c6c8c8227497f1c2fbe732d73b
---
 data/org.freedesktop.portal.Email.xml | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/data/org.freedesktop.portal.Email.xml b/data/org.freedesktop.portal.Email.xml
index 3b99394..39959e3 100644
--- a/data/org.freedesktop.portal.Email.xml
+++ b/data/org.freedesktop.portal.Email.xml
@@ -39,17 +39,17 @@
 
         Note that the default email client for the host will need to support mailto: URIs
         following <ulink url="https://tools.ietf.org/html/rfc2368">RFC 2368</ulink>, with
-        “cc”, “bcc”, “subject” and “body” query keys each corresponding to the email
-        header of the same name, and with each attachment being passed as a “file://”
-        URI as a value in an “attachment” query key.
+        "cc", "bcc", "subject" and "body" query keys each corresponding to the email
+        header of the same name, and with each attachment being passed as a "file://"
+        URI as a value in an "attachment" query key.
 
         For example:
         <programlisting>
         mailto:foo@bar.com,baz@bar.com?cc=ceo@bar.com&amp;subject=Test%20e-mail%20subject&amp;attachment=file://path/to/full/file.txt
         </programlisting>
-        would send a mail to “foo@bar.com”, “baz@bar.com”, with a CC:
-        to “ceo@bar.com”, with the subject “Test e-mail subject”
-        and the file pointed by URI “file://path/to/full/file.txt” as an attachment.
+        would send a mail to "foo@bar.com", "baz@bar.com", with a CC:
+        to "ceo@bar.com", with the subject "Test e-mail subject"
+        and the file pointed by URI "file://path/to/full/file.txt" as an attachment.
 
         Supported keys in the @options vardict include:
         <variablelist>
